src/Raven.Client/Documents/Operations/Attachments/AttachmentDetails.cs

```diff
@@ -64,16 +69,23 @@ public class AttachmentName
         /// The size of the attachment in bytes.
         /// </summary>
         public long Size;
+        public AttachmentFlags Flags;
+        public DateTime? RetireAt;
+        public string Collection;
 
         internal virtual DynamicJsonValue ToJson()
         {
-            return new DynamicJsonValue
+            var json = new DynamicJsonValue
             {
                 [nameof(Name)] = Name,
                 [nameof(Hash)] = Hash,
                 [nameof(ContentType)] = ContentType,
                 [nameof(Size)] = Size
             };
+            json[nameof(Flags)] = Flags.ToString();
```

Comment:
I think .ToString() is not necessary

Reply:
I can remove but its like this in other places as well
https://github.com/ravendb/ravendb/blob/d897a3c1825daf6c4b3bc9b4eecbe57f0ba7320b/src/Raven.Client/Documents/Changes/ChangeNotification.cs#L97